docs/algorithms.md

```diff
@@ -3,15 +3,18 @@ id: algorithms
 title: Algorithm Descriptions
 ---
 
-Captum is a library within which different interpretability methods can be implemented.  The Captum team welcomes any contributions in the form of algorithms, methods or library extensions!  
+Captum is a library within which different interpretability methods can be implemented.  The Captum team welcomes any contributions in the form of algorithms, methods or library extensions!
 
-The algorithms in Captum are separated into three groups, primary attribution, layer attribution and neuron attribution, which are defined as follows:
+The attribution algorithms in Captum are separated into three groups, primary attribution, layer attribution and neuron attribution, which are defined as follows:
 * Primary Attribution: Evaluates contribution of each input feature to the output of a model.
 * Layer Attribution: Evaluates contribution of each neuron in a given layer to the output of the model.
 * Neuron Attribution: Evaluates contribution of each input feature on the activation of a particular hidden neuron.
 
 Below is a short summary of the various methods currently implemented for primary, layer, and neuron attribution within Captum, as well as noise tunnel, which can be used to smooth the results of any attribution method.
 
+Beside attribution algorithms Captum also offers metrics to estimate the trustworthiness of the explanations and model predictions.
```

Comment:
nit: Can we just say trustworthiness of explanations here? This seems to suggest that they can be used to estimate trustworthiness of model predictions as well, although it's primarily the explanations?